spirv_msl.cpp

```diff
@@ -147,10 +148,15 @@ void CompilerMSL::build_implicit_builtins()
 	bool need_multiview = get_execution_model() == ExecutionModelVertex && !msl_options.view_index_from_device_index &&
 	                      (msl_options.multiview || active_input_builtins.get(BuiltInViewIndex));
 	bool need_dispatch_base =
-	    msl_options.dispatch_base && get_execution_model() == ExecutionModelGLCompute &&
-	    (active_input_builtins.get(BuiltInWorkgroupId) || active_input_builtins.get(BuiltInGlobalInvocationId));
+	    (msl_options.dispatch_base && get_execution_model() == ExecutionModelGLCompute &&
```

Comment:
Can this be split up into different statements?

Reply:
Sure.